Packs/Logzio/Integrations/Logzio/README.md

```diff
@@ -0,0 +1,742 @@
+## Overview
+---
+
+Fetch incidents from Logz.io Cloud SIEM to activate Demisto Playbooks
+
+## Use Cases
+---
+
+Integrate with Logz.io Cloud SIEM to automatically remediate security incidents identified by Logz.io and increase observability into incident details. 
+The integration allows Demisto users to automatically remediate incidents identified by Logz.io Cloud SIEM using Demisto Playbooks.
+In addition, users can query Logz.io directly from Demisto to investigate open questions or retrieve the logs responsible for triggering security rules. 
+
+
+## Prerequisites 
+---
+
+1. Logz.io Cloud SIEM - You’ll need to have a Logz.io Cloud SIEM add-on. If you need to add it, please contact support@logz.io. 
+
+2. API Tokens - You’ll need to have active API Tokens for each of the relevant Logz.io accounts. Keep in mind that API tokens are specific to account ID. Your Logz.io Operations accounts and associated Security account have separate API Tokens. 
+
+
+## Logz.io Playbook
+---
+
+Logz.io provides a sample playbook to get you started. You can add as many playbooks as you’ll need to keep increasing your security. 
```

Comment:
please add playbook name here

Reply:
Sample playbook will be delivered for the next release 
so i think we can remove that part from the Readme